web/src/main/java/edu/gemini/lch/web/app/windows/night/BlanketClosuresTable.java

```diff
@@ -139,8 +141,8 @@ public void actionClick(final ActionColumn.Action action, final Object itemId) {
 
     private void endNow(final Object itemId) {
         final BlanketClosure closure = container.getItem(itemId).getBean();
-        if (closure.getStart().isBeforeNow() && closure.getEnd().isAfterNow()) {
-            closure.setEnd(DateTime.now());
+        if (closure.getStart().isBefore(ZonedDateTime.now()) && closure.getEnd().isAfter(ZonedDateTime.now())) {
```

Comment:
I suppose the result of `ZonedDateTme.now()` will be slightly different in these 3 calls. Should we store it in a temporal variable?

Reply:
Absolutely. Good call.